geographic_msgs/GeoPath route
bool isConfirmed